app/views/change_requests/all.slim

```diff
@@ -12,10 +12,20 @@ div.content-container.center
           th Status
       tbody
         -sorted_array = @change_requests.sort {|a,b| a.notebook.title.partition(')')[2] <=> b.notebook.title.partition(')')[2] }
```

Comment:
*ahem*

Reply:
Oh wow, I don't know how I didn't see that